.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - "3.11"
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install UV
+        uses: astral-sh/setup-uv@v5
+
+      - name: Install Python
+        run: uv python install ${{ matrix.python-version }}
```

Comment:
What do you think about using a `.python-version` file in the project root and removing this?

Reply:
For a package I was thinking we would eventually add in other Python versions to prove support for those versions, but while there's no valuable code in here only set to a minimum of 1 version

Reply:
Makes sense, can you specify the python version in the UV setup part?

Reply:
I think actually this is even nicer,

1. Commit `.python-version` for the default developer version
2. Use `with: python-version: ${{ matrix.python-version }}` in CI to override what's in the `.python-version` (see https://github.com/astral-sh/setup-uv?tab=readme-ov-file#python-version)

I just tested with 3.11 and 3.12... uv is "confusingly fast" as others have put it 😮